FRAuth/FRAuth/Authentication/AuthService.swift

```diff
@@ -2,7 +2,7 @@
 //  AuthService.swift
 //  FRAuth
 //
-//  Copyright (c) 2019 ForgeRock. All rights reserved.
+//  Copyright (c) 2020 ForgeRock. All rights reserved.
```

Comment:
I think we are supposed to do a range: `<initial publish year> - <current year>`

Reply:
I just saw the thread that that would be ideal. So will update all files accordingly. Thanks for pointing that out.